Update eliminated players rule for officially supported tournaments (#8560)
ppy/osu-wiki: wiki/Tournaments/Official_support/en.md

The [osu! team](/wiki/People/The_Team) runs a program where they may opt to provide community-run tournaments that abide by a certain set of criteria and expectations with extra support, at their discretion.

> This article was last updated on January 2, 2023. For any queries and clarifications, please consult the `#tournaments` channel on the [osu!dev Discord server](https://discord.com/invite/ppy) or send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh).

Reviewer: Too optimistic. ![](https://static-cdn.jtvnw.net/emoticons/v2/1238492/static/light/1.0)

```suggestion
This article was last updated on January 4, 2023. For any queries and clarifications, please consult the `#tournaments` channel on the [osu!dev Discord server](https://discord.com/invite/ppy) or send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
```

- The tournament series must run no more than four times per year.
- The tournament must operate on a Round of 16 double-elimination format at a minimum, **OR**, have at least 64 players (or 16 teams) participating. Larger scales and formats are permitted.
- The tournament is organised and run by an *experienced team of accomplished and reputable staff*,[^experience] **OR**, the tournament *has been run at least once before without receiving rewards*.

> - Staff members must not participate in the tournament themselves at any point. For our definition on "staff", please refer to the [FAQ](#what-is-considered-staff).

Reviewer: ```suggestion
- Staff members must not participate in the tournament themselves, though they may join after they have been eliminated. For our definition on "staff", please refer to the [FAQ](#what-is-considered-staff).
```

> - Staff members must not participate in the tournament themselves, though they may join after they have been eliminated. For our definition on "staff", please refer to the [FAQ](#what-is-considered-staff).

Reviewer 2: Does this mean that a "player may join as "staff""? If so that needs to be said. Right now it reads as though a staff can join as a player, which I do not believe is the intention.

Reviewer 3: it's even worse since it's not even referring to players. the only thing this sentence is saying is that *staff members may join (the tournament) after being eliminated*, but non-players can't be eliminated so it doesn't make sense.

Author: Yeah okay, I think I was way too tired when I approved this suggestion.
Very confusing indeed. 
Fixed that one.

Author: I might have been way too tired when approving this one.
Changed wording to make more sense.

### Can a player who has been eliminated from play in my tournament join the staff after the fact and not cause problems? {#player-join-staff}

> Yes. Players who have been eliminated from a tournament may be enlisted as staff to assist in any non-managerial position.

Reviewer 2: The previous text was very confusing so I agree the new version is better. But what is "non-managerial"? It's probably best to clarify since I can't understand this.

Author: This one was mostly turned this way as non-managerial position could differ depending on the structure of the tournament.
We probably could add most roles that are counted that most tournaments have, and just tell anyone with a different role to contact if required, as I'd assume they wouldn't be in this much cases.

Reviewer 2: Why not link to the same anchor as the other one above?

Author: Discussed with PooN a bit, we decided to add the list of staff members that is above that question as it makes more sense anyway, since other roles aren't gonna be coming up this often.

> Yes. Players who have been eliminated from a tournament may be enlisted as staff to assist to assist in any non-managerial position.

Reviewer 4: ```suggestion
Yes. Players who have been eliminated from a tournament may be enlisted as staff to assist in any non-managerial position.
```

> Yes. Players who have been eliminated from a tournament may be enlisted as staff to assist as a referee, mappool selector, custom mapper, and/or a playtester.

Reviewer: Our poll stated "every non-admin/non-managerial position". I think this covers different roles depending on the structure of the tournaments; perhaps it's better to phrase it the other way around?

```suggestion
Yes. Players who have been eliminated from a tournament may be enlisted as staff to assist to assist in any non-managerial position.
```

Author: Would work better indeed, wasn't too sure. Thanks!

> Yes. Players who have been eliminated from a tournament may be enlisted as staff to assist in any non-managerial position (referees, mappool selectors, mappool playtesters, custom mappers, judges, etc.) This list is not exhaustive. If you have any doubts about any role not noted here, please contact [tournaments@ppy.sh](mailto:tournaments@ppy.sh).

Reviewer: Un-needed repetition (though unsure if I'm doing this linking right, a wiki guy will have to confirm)

```suggestion
Yes. Players who have been eliminated from a tournament may be enlisted as staff to assist in any of the non-managerial positions [mentioned above](#what-is-considered-staff).
```

Reviewer 3: the current text also contradicts itself. i've bolded those parts:

> To clarify, we consider organisers, **referees, mappool selectors, mappool playtesters, custom mappers, judges, or any other form of managerial position** directly involved in the maintenance and operation of the tournament/contest to be a part of its staff.

> any **non-managerial position (referees, mappool selectors, mappool playtesters, custom mappers, judges**, etc.)

---

if i read your suggestion correctly, then

> Players who have been eliminated from a tournament may be enlisted as staff to assist in any of the non-managerial positions [mentioned above](#what-is-considered-staff)

could really only sort of be interpreted to refer to

> Commentators, streamers, and graphic designers are designated as **helpers**, and may participate in tournaments/contests freely so long as they are not directly involved in any organisational aspect of the tournament/contest.

which would mean the same thing as before this pr.

this page draws the distinction between helpers and staff but doesn't clarify what a *managerial* and *non-managerial* is. i believe that was peppy's point as well.

Author: This should be changed indeed.
Any mangerial role is basically anything above these roles, so mostly organizers/admins really. I don't remember any other role that would be above the other roles noted out, so we could change it from this:

> any non-managerial position (referees, mappool selectors, mappool playtesters, custom mappers, judges, etc.)

to this: 

> any non-organizational position (Organizers, admins...)

I feel like these would be the only ones incorporated, if anyone can  let me know if I did it would be nice.

### Can a player who has been eliminated from play in my tournament join the staff after the fact and not cause problems? {#player-join-staff}

> Yes. Players who have been eliminated from a tournament may be enlisted as staff to assist in any non-organizational position (Organizers, admins..) This list is not exhaustive. If you have any doubts about any role not noted here, please contact [tournaments@ppy.sh](mailto:tournaments@ppy.sh).

Reviewer 3: ```suggestion
Yes. Players who have been eliminated from a tournament may be enlisted as staff to assist in any non-organisational position (i.e. no organisers, admins, etc.). If you have any doubts about any role not noted here, please contact [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
```
